tests/amountClaim/pages/submitted.js

```diff
@@ -3,9 +3,11 @@
 
 const user = require('../../../test-data').user
 const verifyPageData = require('../../../test-data').verifyPageData
-let assert = require('assert')
-
-let I
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+chai.use(chaiAsPromised)
+let I, expect
```

Comment:
shouldn't do multiple assigns in one line it can be confusing,
`const expect = chai.expect` should be fine to do